src/agent.spec.ts

```diff
@@ -0,0 +1,549 @@
+import { Finding, ethers, FindingSeverity, FindingType } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress, createTransactionHash } from "forta-agent-tools/lib/utils";
+import agent from "./agent";
+
+const { handleTransaction } = agent;
+
+import {
+  NETHERMIND_DEPLOYER_ADDRESS,
+  AGENT_REGISTRY_ADDRESS,
+  CREATE_AGENT_ABI,
+  UPDATE_AGENT_ABI,
+  DISABLE_AGENT_ABI,
```

Comment:
Requesting you to declare mock constants in the test file rather than using the ones that would be used in production.

That's because the bot could seem like it is working even if one of these is wrong. If it is tested with an incorrect constant and it will rely on that same constant. Then it saying it works would mean more that it is "consistent in it being broken".